### ISSUE TITLE
Add reduced text to components via the extension

### ISSUE THREAD

**Issue by RobertPeek** [deltanet/adapt-audio#43]
The reduced text needs to be changed on the components title and body through the extension rather than the individual components.

Also a full review of the config needs finalising so the extension can be added to the authoring tool.

**Comment by dancgray** [comment]
I think audio:changeText should pass through the value for Adapt.audio.textSize so that the plugins do not need to reference this value e.g: 

https://github.com/deltanet/adapt-contrib-mcq-audio/blob/authoringTool/js/adapt-contrib-mcq-audio.js#L32

adapt-audio should also control when this is called so plugins do not have to set text size on render:
https://github.com/deltanet/adapt-contrib-mcq-audio/blob/authoringTool/js/adapt-contrib-mcq-audio.js#L97

Basically decouple components from adapt-audio :)

**Comment by dancgray** [comment]
Fixed
